app/controllers/presentations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PresentationsController < ApplicationController
+  def show
+    @presentation = Presentation.find(params[:id])
+  end
+
+  def new
+    @presentation = Presentation.new
+  end
+
+  def create
+    @presentation = Presentation.new(presentation_params)
+
+    if @presentation.save
+      flash[:notice] = 'Presentation was successfully created'
+      redirect_to @presentation
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def presentation_params
+    # binding.pry
```

Comment:
What if future devs need to use pry, but forget how to do it?!

I wish comments had a facepalm emoji